pandas/io/formats/format.py

```diff
@@ -179,15 +180,24 @@ def __init__(self, series, buf=None, length=True, header=True, index=True,
 
     def _chk_truncate(self):
         from pandas.core.reshape.concat import concat
+        min_rows = self.min_rows
         max_rows = self.max_rows
+        # if min_rows is None, follow value of max_rows
+        if max_rows and not min_rows:
+            min_rows = max_rows
+        # if both are set, min_rows is minimum of both
+        if min_rows and max_rows:
+            min_rows = min(min_rows, max_rows)
+        # truncation determined by max_rows, actual truncated number of rows
+        # used below by min_rows
         truncate_v = max_rows and (len(self.series) > max_rows)
         series = self.series
         if truncate_v:
-            if max_rows == 1:
-                row_num = max_rows
-                series = series.iloc[:max_rows]
+            if min_rows == 1:
+                row_num = min_rows
+                series = series.iloc[:min_rows]
             else:
-                row_num = max_rows // 2
+                row_num = min_rows // 2
```

Comment:
maybe should just do a 

```
min_rows = min_rows or 0
```

at the beginning of the function to ensure this is an integer

Reply:
Not fully sure what you mean. The floor division here is because min_rows can be an uneven number.